Add localization for Japanese, Chinese and Korean

Xzonn opened the pull request:
Thanks for your nice work! I think it would be better if this tool can be localized into more languages. So I added official names for items and monsters in Japanese, Simplified Chinese, Traditional Chinese, and Korean.

Xzonn committed:
Add localization for Japanese, Chinese and Korean

Xzonn committed:
Forget to handle English

PippleCultist commented:
This looks really good. Before I merge it, can you add a new dropdown box that lets the user choose the language mode manually as well? You will probably need to create new arrays for the translated text so that the original arrays can be reused to translate. Also, can you change the indentation from spaces to tabs? Thanks for the great work!

Xzonn committed:
Change the indentation style

Xzonn committed:
Add a dropdown box for choosing the language

Xzonn committed:
Update Korean translation

Xzonn commented:
> This looks really good. Before I merge it, can you add a new dropdown box that lets the user choose the language mode manually as well? You will probably need to create new arrays for the translated text so that the original arrays can be reused to translate. Also, can you change the indentation from spaces to tabs? Thanks for the great work!

Thanks for your suggestions! I have added a dropdown menu and modified the code format.
